static/js/biomedical/disease/data_processing_utils.ts

```diff
@@ -326,3 +326,28 @@ export function getDiseaseCommonName(data: GraphNodes): string {
     return formattedDiseaseName;
   }
 }
+/**
+ * Checks whether the stat var for medical condition of disease exists
+ * @param data
+ * @returns boolean indicating if desired stat var exists
+ */
+export function doesDiseasePrevalenceIDexist(data: GraphNodes): boolean {
+  // sets the default value of the boolean as false
+  if (!data) {
+    return false;
+  }
+  for (const neighbour of data.nodes[0].neighbors) {
+    if (neighbour.property !== "medicalCondition") {
+      continue;
+    }
+    // check for null or non-existent property values
+    if (_.isEmpty(neighbour.nodes)) {
+      continue;
+    }
+    const medicalCondition = neighbour.nodes[0].value;
+    if (!_.isEmpty(medicalCondition)) {
+      return true;
+    }
+    return false;
```

Comment:
`return false` should probably be outside the for loop?

Reply:
Yes, good catch! Moved it outside the for loop!